src/components/CommentAdder.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
```

Comment:
This ticket isn't for posting comments, please don't add things we aren't here to feedback on.